Sample of Get Element CSS Value

Peyton88 opened the pull request:
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

Peyton88 committed:
Sample of Get Element CSS Value

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=527) <br/>All committers have signed the CLA.

harsha509 reviewed (changes requested):
Hi @Peyton88 ,

Thanks for the PR!

get Element css value actually returns a value of computed css style property of an element. we have WebDriver method like `getCssValue` in JS. The code sample you've provided is for `getAttribute`

i've updated doc with a code sample for JS. https://www.selenium.dev/documentation/fr/webdriver/web_element/#get-element-css-value

can you please update the code samples accordingly.

Thank you !

Peyton88 commented:
> Hi @Peyton88 ,
> 
> Thanks for the PR!
> 
> get Element css value actually returns a value of computed css style property of an element. we have WebDriver method like `getCssValue` in JS. The code sample you've provided is for `getAttribute`
> 
> i've updated doc with a code sample for JS. https://www.selenium.dev/documentation/fr/webdriver/web_element/#get-element-css-value
> 
> can you please update the code samples accordingly.
> 
> Thank you !

Thanks. I got a misunderstanding.
I've updated my commit.

harsha509 commented on docs_source_files/content/webdriver/web_element.en.md:
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
+driver.get 'https://www.example.com'
+
+# Retrieves the computed style property 'color' of linktext
+cssValue = driver.find_element(:link_text, 'More information...').css_value('color');
```
```suggestion
cssValue = driver.find_element(:link_text, 'More information...').css_value('color')
```

harsha509 commented on docs_source_files/content/webdriver/web_element.en.md:
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
-// Please provide a pr for the code sample
+
+// Navigate to Url
+driver.get('https://www.example.com');
```
```suggestion
driver.get('https://www.example.com')
```

harsha509 commented on docs_source_files/content/webdriver/web_element.en.md:
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
+driver.get('https://www.example.com');
+
+# Retrieves the computed style property 'color' of linktext
+cssValue = driver.findElement(By.LINK_TEXT, "More information...").value_of_css_property('color');
```
```suggestion
cssValue = driver.findElement(By.LINK_TEXT, "More information...").value_of_css_property('color')
```

harsha509 commented on docs_source_files/content/webdriver/web_element.en.md:
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
-// Please provide a pr for the code sample
+
+# Navigate to Url
+driver.get('https://www.example.com');
```
```suggestion
driver.get('https://www.example.com')
```

harsha509 commented on docs_source_files/content/webdriver/web_element.en.md:
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
+driver.get('https://www.example.com');
+
+// Retrieves the computed style property 'color' of linktext
+val cssValue = driver.findElement(By.linkText("More information...")).getCssValue('color');
```
```suggestion
val cssValue = driver.findElement(By.linkText("More information...")).getCssValue('color')
```

harsha509 reviewed (changes requested):
hi @Peyton88 ,

Thank you for making the changes. and apologies for reverting it, there are couple of things required to get this pr merged !

1. Can you please fill/complete the above checklist.
2. Semicolons are not required in few language bindings like kotlin, ruby, python can you please look into the suggested changes
3. Can you please make the same changes in other translated pages too like (es,fr,zh-cn).

Thank you!

Peyton88 commented:
> hi @Peyton88 ,
> 
> Thank you for making the changes. and apologies for reverting it, there are couple of things required to get this pr merged !
> 
> 1. Can you please fill/complete the above checklist.
> 2. Semicolons are not required in few language bindings like kotlin, ruby, python can you please look into the suggested changes
> 3. Can you please make the same changes in other translated pages too like (es,fr,zh-cn).
> 
> Thank you!

Hi @Harsha509 ,

Thanks for your kindly reminding.
Please see my updates.

harsha509 reviewed (approved):
Thank you @Peyton88 !

harsha509 committed:
Merge branch 'dev' into dev

harsha509 commented:
deploy-preview looks good !

harsha509 commented:
Congratulations on your first contribution @Peyton88  🎉  🎉  🎉  🎉  🎉

Peyton88 commented:
> Congratulations on your first contribution @Peyton88 🎉 🎉 🎉 🎉 🎉

Thanks you~
